### PR TITLE
Update Boeing-Employees-Credit-Union.xml

### PR DESCRIPTION
**Pull request by minigeek**
Removes Typo which broke all CSS and JS on the site.

### CODE REVIEW

File: src/chrome/content/rules/Boeing-Employees-Credit-Union.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 	<!--	Some pages redirect to http.  There may be more that don't.	-->
 	<rule from="^http://www\.becu\.org/($|contact-us\.aspx|css/|Default\.aspx|flash/|images/|js/|mobile-online-banking/|pdfsource/|who-is/becu-is-you\.aspx)"
-		to="https://www.becu.org/$1s/" />
+		to="https://www.becu.org/$1/" />
```
Comment (reedy) [inline src/chrome/content/rules/Boeing-Employees-Credit-Union.xml]: Is the trailing / still wanted?

https://www.becu.org/who-is/becu-is-you.aspx works

https://www.becu.org/who-is/becu-is-you.aspx/ gives a 404

Comment (minigeek) [inline src/chrome/content/rules/Boeing-Employees-Credit-Union.xml]: You're right, fixed

**Comment by jsha** [comment]
Per the new [ruleset testing](https://github.com/EFForg/https-everywhere/blob/master/ruleset-testing.md) requirements and [ruleset style](https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md) guide, can you change this rule to list specify hosts in the target hosts, rather than wildcards?

Thanks,
Jacob

**Comment by minigeek** [comment]
Nah, I just saw a bug and figured I'd help out, if you don't want to pull please feel free to close.

**Comment by jsha** [comment]
Sorry, you got caught up in a bulk edit as part of a testing process we're going through. It looks like this ruleset is currently disabled in master, so we can merge it without breaking our tests. Thanks for your contribution!

**Comment by minigeek** [comment]
No problem. Thanks for understanding!

**Pull request merged**